#!/usr/bin/env python
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("rotation_center_frame", str_t, 0, "The TF frame used to publish the center of rotation.", "/rotation_center")
gen.add("rotating_frame", str_t, 0, "The TF frame used to publish the rotating objects", "/rotating_objects")

exit(gen.generate("object_tracker", "rotation_tracker", "RotatingObjectTracker"))
